recipe-server/client/control_new/state/extensions/actions.js

```diff
@@ -74,3 +86,15 @@ export function updateExtension(pk, extensionData) {
     });
   };
 }
+
+export function saveExtensionListingColumns(columns) {
+  return async dispatch => {
+    await localForage.setItem('extension_listing_columns', columns);
```

Comment:
Technically, this `await` could be removed, since the UI doesn't need to be hung up on waiting for localForage.

Also, I noticed there's only a `save` and no `load` function for these columns - is that for another PR?